ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/ScheduledProtocolSpec.java

```diff
@@ -14,21 +14,11 @@
  */
 package org.hyperledger.besu.ethereum.mainnet;
 
-/** Tuple that associates a {@link ProtocolSpec} with a given block level starting point */
-public class ScheduledProtocolSpec {
-  private final long block;
-  private final ProtocolSpec spec;
-
-  public ScheduledProtocolSpec(final long block, final ProtocolSpec spec) {
-    this.block = block;
-    this.spec = spec;
-  }
-
-  public long getBlock() {
-    return block;
-  }
+/**
+ * Tuple that associates a {@link ProtocolSpec} with a given block number or timestamp level
+ * starting point
+ */
+public interface ScheduledProtocolSpec {
```

Comment:
What does this type represent now? It's no longer a tuple with a block or timestamp, it just has a spec() method.

Reply:
Have made this a record that is used by both types of schedule